pages/docs/tools/cli/usage.md

```diff
@@ -29,7 +29,7 @@ $ npm install -g @asyncapi/cli
 $ asyncapi COMMAND
 running command...
 $ asyncapi (--version)
-@asyncapi/cli/0.54.3 linux-x64 node-v18.17.1
+@asyncapi/cli/0.54.4 linux-x64 node-v18.17.1
```

Comment:
Why these changes are done in this PR?

Reply:
i had to use git pull because of some reason and it came with it